mlir/test/CMakeLists.txt

```diff
@@ -50,6 +50,14 @@ if(NOT ROCMLIR_DRIVER_RANDOM_DATA_SEED STREQUAL "none")
   set(MLIR_RANDOM_DATA "-rand ${ROCMLIR_DRIVER_RANDOM_DATA_SEED} ")
 endif()
 
+add_custom_target(rocmlir-common-python-test-utils
```

Comment:
Please make this depend on the contents of that directory (or, really, do an `add_subdirectory` here and then copy the files) so it's not an unconditional copy when nothing's been changed?

Reply:
Hi @krzysz00 , can you have you look at the changes. I guess it looks better now